src/app/services/program.service.ts

```diff
@@ -10,7 +10,8 @@ export class Episode {
     public noImpressionUrl?: string,
     public keywords?: string[],
     public warning?: string,
-    public arrangement?: {id: string, isOriginal: boolean}[]
+    public arrangement?: {id: string, isOriginal: boolean}[],
+    public categories?: string[]
```

Comment:
So it looks like the `keywords` will also include "adfree" ... but they only get set on the Episode _after_ we make a debug request to Dovetail.  So that's not helpful.

Yeah, parsing the XML categories separately should work.